Tests failing but CI checks reported as successfull

## Issue description
I'm working on PR #782. My PR is not completed and the tests `test_segment_volume_2d` and `test_segment_volume_2d_with_patches` on github are failing at the moment (which is normal).

However, the checks from the new Github Action CI on multi-platform are reported as successfull, even though they are not. 
Example in `Run tests on all platforms / Matrix Test of Python 3.6 on macos-latest (pull_request)`:
```
=========================== short test summary info ============================
FAILED testing/functional_tests/test_segment_volume.py::test_segment_volume_2d
FAILED testing/functional_tests/test_segment_volume.py::test_segment_volume_2d_with_patches[center_crop0]
FAILED testing/functional_tests/test_segment_volume.py::test_segment_volume_2d_with_patches[center_crop1]
=========== 3 failed, 196 passed, 370 warnings in 495.46s (0:08:15) ============
```
Also, the check `Matrix Test of Python 3.8 on macos-11` seems to be stuck.

Tagging @dyt811 so he is aware.

--- a/.github/workflows/run_tests.yml
+++ b/.github/workflows/run_tests.yml
@@ -30,7 +30,7 @@ jobs:
     # Default shell for ALL subsequent steps.
     defaults:
       run:
-        shell: bash {0}
+        shell: bash
 
     # Defining matrix for OS and Python
     strategy:

